guppylang/prelude/builtins.py

```diff
@@ -87,6 +87,9 @@ def __new__(x): ...
     @guppy.hugr_op(builtins, logic_op("Or", [tys.TypeArg(tys.BoundedNatArg(n=2))]))
     def __or__(self: bool, other: bool) -> bool: ...
 
+    @guppy.hugr_op(builtins, DummyOp("Xor"))
+    def __xor__(self: bool, other: bool) -> bool: ...
```

Comment:
The logic extension doesn't define Xor yet. See https://github.com/CQCL/hugr/issues/1418